### PR TITLE
Add `BuyWidget` non-sale state

### PR DESCRIPTION
**Pull request by chinins**
## Description

This PR add "non-sale" state to the `BuyTokens` widget - UI only.

**New stuff** ✨

- add html & css for the new state

**Changes** 🏗

- update css & html in the `BuyTokens`

## TODO

I would separate the form from the `BuyTokens` component as it's already too big - once Raul finishes his PR, don't want to cause too many merge conflicts.

Resolves DEV-245

Screenshot 2021-07-27 at 16 36 04

### CODE REVIEW

**Comment by rdig** [comment]
Please hold off reviewing this until later today after I finish #2445 

I've made some changes there that need to be accounted for _(I think)_ in here as well

**Review by alicjakujawa — CHANGES_REQUESTED** [review]
All of the saga placeholder code and translations for it shouldn't be done here. It was UI only.

**Comment by chinins** [comment]
> All of the saga placeholder code and translations for it shouldn't be done here. It was UI only.



> All of the saga placeholder code and translations for it shouldn't be done here. It was UI only.

My branch is only UI, it's just Raul changed some of his stuff and I need to rebase.

**Comment by rdig** [comment]
I've merged my #2445 so this can now be rebased, then reviewed

File: src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx [hunk 1]
```diff
@@ -85,6 +104,10 @@ const validationSchema = (userBalance: number) =>
 const BuyTokens = ({ colony: { colonyAddress }, disabled }: Props) => {
   const { username, ethereal, walletAddress } = useLoggedInUser();
 
+  /* To add proper states later */
+  const isSale = false;
```
Comment (alicjakujawa) [inline src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx:108]: Actually i think we already can find out (with Raul changes) if there is a sale ongoing.

File: src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx [hunk 2]
```diff
@@ -230,186 +255,224 @@ const BuyTokens = ({ colony: { colonyAddress }, disabled }: Props) => {
...
                                 parseFloat(currentSalePrice)
-                              : '0'
-                          }
-                        />
-                      </div>
-                    )
-                  }
+                              ).toFixed(2)
+                            : ''}
+                        </div>
```
Comment (alicjakujawa) [inline src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx:394]: Do we need this div if we dont have values.amount?

**Review by alicjakujawa — APPROVED** [review]
All good here, just 2 suggestions!

File: src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx (hunk 2 above)
Comment (chinins) [reply src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx:394]: That's from Raul's PR, so I guess we do.

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me! I would like this to be merged before #2622 so that I can connect the dialogs with the `Get whitelisted` button you have here in that PR.

Nice work :100:

File: src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx (hunk 1 above)
Comment (chinins) [reply src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx:108]: I had a look and also checked with Raul and it's not worth it for now. There is only partial logic there so no point in trying to finish it up now.

**Pull request merged**